geomesa-tools-common/conf/geomesa-default.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we rename this to `geomesa-site.xml.template`? otherwise it's not clear that this file isn't actually used for anything.